src/components/Header/Header.jsx

```diff
@@ -22,32 +23,22 @@ export class _Header extends React.Component {
   constructor(props) {
     super(props);
 
-    this.state = { openMenu: false };
+    this.state = {
+      openMenu: false,
+      hideLoginLink: false,
```

Comment:
[question] I might be missing it, but where are `state.hideLoginLink` & `state.hideLanguageSwitch` being used? If they are not being used consider removing them from the state object

Reply:
Ahhh.... that looks like it ended up getting leftover from my original display of the headers on the docs. I will clean that up!